crates/iroha/Cargo.toml

```diff
@@ -85,7 +85,8 @@ iroha_genesis = { workspace = true }
 iroha_test_samples = { workspace = true }
 iroha_test_network = { workspace = true }
 iroha_executor_data_model = { workspace = true }
-executor_custom_data_model = { version = "=2.0.0-rc.1.0", path = "../../wasm_samples/executor_custom_data_model" }
+iroha_multisig_data_model = { workspace = true }
```

Comment:
we might want to make `multisig_data_model` a regular dependency of `iroha` and re-export it so that users can interact with multisig without additional complexity